generator/src/main.rs

```diff
@@ -1576,6 +1578,7 @@ impl Parser {
     fn compute_meta(&self, name: &str, s: &Struct) -> StructMeta {
         let mut out = StructMeta::default();
         for member in &s.members {
+            out.has_function_pointer |= member.ty.starts_with("PFN") || member.ty == "LUID";
```

Comment:
I don't think `LUID`s are function pointers. Maybe rename to `has_unprintable`?

Reply:
Done.